queues/fair.go

```diff
@@ -87,7 +102,7 @@ var scriptFairDone = valkey.NewScript(1, luaFairDone)
 
 // Done marks the passed in task as complete. Callers must call this in order
 // to maintain fair workers across orgs
-func (q *Fair) Done(ctx context.Context, vc valkey.Conn, owner string) error {
+func (q *Fair) Done(ctx context.Context, vc valkey.Conn, owner OwnerID) error {
```

Comment:
I thought we'll need to pass the ID to Done as well but for now we can use that in the logs only